libdnf/transaction/db/trans_item.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright (C) 2017-2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "trans_item.hpp"
+
+#include "repo.hpp"
+
+#include "libdnf/transaction/transaction.hpp"
+#include "libdnf/transaction/transaction_item.hpp"
+
+
+namespace libdnf::transaction {
+
+
+void transaction_item_select(libdnf::utils::SQLite3::Query & query, TransactionItem & ti) {
+    ti.set_id(query.get<int64_t>("id"));
```

Comment:
`query.get<int64_t>("id")` looks strange to me.

Reply:
Could you be more specific?
Does it mean you don't understand the code just by looking at it and it requires a comment explaining it?